app/build.gradle

```diff
@@ -14,8 +14,8 @@ android {
         applicationId "org.openobservatory.ooniprobe"
         minSdkVersion 9
         targetSdkVersion 25
-        versionCode 4
-        versionName "1.1.2"
+        versionCode 5
+        versionName "1.1.3-dev"
```

Comment:
It's good that this is while we are developing `-dev`. Remember to bump this up to `1.1.3` when merging.